### DISCUSSION TITLE
Bricked bootloader

### DISCUSSION THREAD

**Discussion by ElectricPulse** [NicoHood/HoodLoader2#118] [General]
I have the following configuration: Arduino Uno SMD Clone, Arduino Mega 2560 (the real deal)
I have it connected like in the pictures, the flashing succeeds.
After double shorting the GND and RESET pins on the mega board. Nothing happens. No usb device appears in dmesg output or on windows.
I have also tried the self flashing method before, but I didnt have such a small capacitator as in the guide, I could only try it with a 470uF (I didnt discharge it before hand - I wonder if that might have killed the board if it was indeed charged).
Rotating the connections by 180 degrees didnt help.
The chip on the mega is indeed a 16u2.
None of my two clones have the bootloader header so I cannot try that.
I tried powering the mega externally, no change.
Any other troubleshooting steps I should try?

If it is relevant, I flashed HoodLoader1 to the device, then I replaced it with the stock firmware.

**Comment by NicoHood** [comment]
So if you only flashed the firmware (not the bootloader), then you should be able to recover. You did the flash via DFU? I guess you should be able to enter DFU again. If not, your MCU is possibly broken, maybe you connected something wrong?

**Reply by ElectricPulse** [reply]
I only uploaded the installation project to the arduino uno and tried to connect the uno to the mega. Nothing more. And no the device really cant be accessed in any way.
